Makefile

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Doing this in cmake would be better to stick a single build system throughout the project